tools/tests/BFTest.sh

```diff
@@ -0,0 +1,89 @@
+check_status()
```

Comment:
check_status is used after every command. It essentially does what the -e flag on the bash shebang does, but it prints out a statement to the serial log file that the Jenkins script is looking for. If I just did the -e flag, the Jenkins script would stall out if there was an error in a command. I added the copyright header and the shebang to the top of the script.